terraform/modules/genericservicewithstoragemount/main.tf

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
I think we can rename the directories to have '_'
then your directory would look better

Reply:
done